packages/commonwealth/client/scripts/views/components/AddToHomeScreenPrompt/AddToHomeScreenPrompt.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useState } from 'react';
+import { CWShareIcon } from '../component_kit/cw_icons/cw_custom_icons';
+import './AddToHomeScreenPrompt.scss';
+
+interface AddToHomeScreenPromptProps {
+  isIOS: boolean;
+  isAndroid: boolean;
+}
+
+const AddToHomeScreenPrompt: React.FC<AddToHomeScreenPromptProps> = ({
+  isIOS = false,
+  isAndroid = false,
+}) => {
+  const [showPrompt, setShowPrompt] = useState(true);
+
+  useEffect(() => {
+    const hidePromptTime = localStorage.getItem('hidePromptTime');
+    if (hidePromptTime && new Date().getTime() < Number(hidePromptTime)) {
+      setShowPrompt(false);
+    }
+  }, []);
+
+  const hidePromptForNDays = () => {
+    const maxDays = 30; // Maximum number of days to hide the prompt
+    let n = Number(localStorage.getItem('hidePromptDays')) || 1; // Get the current number of days from local storage, default to 1 if not set
+    n = n * 2 > maxDays ? maxDays : n * 2; // Double the number of days, but cap at maxDays
+    const hideUntil = new Date().getTime() + n * 24 * 60 * 60 * 1000;
+    localStorage.setItem('hidePromptTime', hideUntil.toString());
+    localStorage.setItem('hidePromptDays', n.toString()); // Store the new number of days in local storage
```

Comment:
those comments look like GPT generated. We should use comments sparingly to explain why we do this, not what every single line does.

Reply:
also, put `hidePromptTime` and `hidePromptDays` to const and resuse it

Reply:
@masvelio so for the hidePromptTIme and hidePromptDays would you want me to just extract the strings to the const to avoid magic strings?

Reply:
have done that in the latest push